README.md

```diff
@@ -1,54 +1,106 @@
-        "ta_review_url": "/courses/{course_id}/group_work/{group_id}?activate_block_id={activate_block_id}",
-        "access_dashboard_for_all_orgs_groups": ["super_admin"],
-        "access_dashboard_groups": ["client_admin"],
-        "ta_roles": ["assistant"]
-      }
-    },
+## Setup
+
+In order to use Group Project XBlock v2 in a course, XBlock must be installed into LMS environment and enabled in 
+course's advanced settings. Also, Group Project XBlock uses a number of instance-specific configuration variables, so 
+setting them is considered a part of setup process.
+
+### Installing XBlock into LMS
+
+If you're using edx-solutions/edx-platform fork, Group Project v2 XBlock is already installed. Otherwise, installing 
+Group Project XBlock v2 is done the very same way other XBlocks are installed:
+
+1. Open terminal (or shell) to the LMS instance.
+2. Switch to LMS virtualenv
+3. pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@ **version_hash** #egg=xblock-group-project-v2
+
+*version_hash* is the hash of the git commit. By the time of writing, latest stable version was `4322ca8092c5385d5602143077e388017fcb1249`,
+so the above command looked like this:
+
+    pip install -e git+https://github.com/open-craft/xblock-group-project-v2.git@4322ca8092c5385d5602143077e388017fcb1249#egg=xblock-group-project-v2
+    
+### Setting configuration variables
+
+There are two sources of configuration variables: django settings and XBlock-specific settings available 
+through [SettingsService][settings-service]. Both types of settings can be set via LMS environment file `lms.env.json`
+or (not recommended) directly in instance Django settings.
 
-These settings have following meanings:
+[settings-service]: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/services.py#L7
 
-* ``dashboard_details_url``: url pattern used to generate details url in the
-  dashboard. It uses following parameters in ``str.format`` style:
+The following django settings are used:
 
-  * ``program_id``: Id of course group this activity belong to
-  * ``course_id``: Id of course this activity belong to
-  * ``project_id``: Id of group project to show
-  * ``activity_id``: Xblock activity to show
+* `EDX_API_KEY`: string - must contain a edX API Key. As this XBlock uses edX API extensively, failure to provide 
+    the key will prevent Group Project XBlock v2 from operating normally.
+* `BASE_DIR`: string - base django instance directory. Used by Submission XBlocks as part of file storage location if
+    local file storage is used
+* `API_LOOPBACK_ADDRESS`: URL - (optional) should contain base URL of LMS API. Default: http://127.0.0.1:8000
+* File upload features piggyback on django file storage mechanism, in order to store files, a file storage mechanism 
+    should be configured *Note:* existing production instances use S3 as file storage; using local file storage is not 
+    confirmed.
```

Comment:
@itsjeyd it means that we never invested any time into actually trying to use local storage. It does not work out of the box, as URLs returned by default file storage mechanism are not forwarded to LMS through nginx. So, most likely there's a way to use local storage, but we neither confirmed it works, nor we have any proofs it does not.